### PR TITLE
Fix Linux CI to actually run inside a docker container

### PR DESCRIPTION
**Pull request by safern**
Current CI was not running under a docker container. Use a different approach using a vsts variable to run under the container.

cc: @eerhardt @Ivanidzo4ka

### CODE REVIEW

**Comment by safern** [comment]
You can now see that it initializes the container and stops it at the beginning and end of the Linux phases.

File: build/ci/phase-template.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
   queue: {}
 
 phases:
   - phase: ${{ parameters.name }}
     variables:
       _buildScript: ${{ parameters.buildScript }}
       _phaseName: ${{ parameters.name }}
+      # if dockerImage is not equal to '' then run under docker container
+      ${{ if ne(parameters.dockerImage, '') }}:
+        _PREVIEW_VSTS_DOCKER_IMAGE: ${{ parameters.dockerImage }}
```
Comment (eerhardt) [inline build/ci/phase-template.yml]: @chcosta - do you have any idea how long we will need to continue using this "preview" feature in VSTS? And when a replacement will be possible?

**Comment by safern** [comment]
@Ivanidzo4ka if you rebase your PR now, you should be able to run the tests :smile:

**Comment by Ivanidzo4ka** [comment]
@safern  it's finally working! Thank you!

**Pull request merged**